lib/browser/objects-registry.js

```diff
@@ -20,6 +20,7 @@ class ObjectsRegistry {
   add(webContents, obj) {
     // Get or assign an ID to the object.
     let id = this.saveToStorage(obj);
+    this.storage[id].count++;
```

Comment:
There are two cases:

```
b1 = remote.getCurrentWindow()
b1.isMinimized()
b1 gets garbage collected
b2 = remote.getCurrentWindow()
b2.setSize()
b2 gets garbage collected
```

In this case b1 and b2 are two different instances, and the callback of  `setDestructor()` will be called twice.

Another case:

```
b1 = remote.getCurrentWindow()
b1.isMinimized()
b2 = remote.getCurrentWindow()
b2.setSize()
b1 === b2
b1/b2 gets garbage collected
```

However in this case b1 and b2 are actually the same object and the callback of  `setDestructor()` will only be called once.

I think the bug is, in the first case, b1 is garbaged collected before b2 is created, but the callback of `setDestructor()` gets called after b2 gets created, so when `b2.setSize()` is called exception would happen.